scripts/hooks/prepare-commit-msg

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
```

Comment:
User `env` and `bash`:
```suggestion
#!/usr/bin/env bash
```
We're not going to be running this on some ancient AIX machines.